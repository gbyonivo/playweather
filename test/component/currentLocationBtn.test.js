import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import CurrentLocationBtn from '../../src/components/currentLocationBtn';

describe('Button component', () => {
  it('renders correctly when isLoading is true', () => {
    const actual = shallow(
      <Provider>
        <CurrentLocationBtn
          isFetching={false}
          fetchData={() => { }}
        />
      </Provider>
    );
    expect(actual).toMatchSnapshot();
  });
});