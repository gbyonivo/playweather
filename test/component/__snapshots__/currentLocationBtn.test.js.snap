// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders correctly when isLoading is true 1`] = `
<Geolocated(CurrentLocationBtn)
  fetchData={[Function]}
  isFetching={false}
/>
`;
